templates/connect-inject-deployment.yaml

```
@@ -85,8 +85,10 @@ spec:
                 -envoy-image="{{ .Values.global.imageEnvoy }}" \
                 -consul-k8s-image="{{ default .Values.global.imageK8S .Values.connectInject.image }}" \
                 -listen=:8080 \
+                {{- if .Values.connectInject.healthChecks.enabled }}
                 -enable-health-checks-controller={{ .Values.connectInject.healthChecks.enabled }} \
```

```suggestion
                -enable-health-checks-controller=true \
```

This would match 
```
                {{- if .Values.connectInject.centralConfig.enabled }}
                -enable-central-config=true \
                {{- end }}
```

and

```
                {{- if .Values.global.enableConsulNamespaces }}
                -enable-namespaces=true \
```

---

I have no idea how that got in there.. thx for the catch!